src/route.js

```diff
@@ -93,12 +93,12 @@ module.exports = class Route {
 
       case Restypie.RouterTypes.KOA_ROUTER:
         /* istanbul ignore next */
-        return function * restypieRouteHandler (next) {
-          if (this.request.body) this.req.body = this.request.body;
-          this.req.params = this.params; // Copy params so that we don't have to parse them
-          this.req.query = this.query;
-          this.state.bundle = new Restypie.Bundle({ req: this.req, res: this.res, url });
-          yield next;
+        return async function restypieRouteHandler (ctx, next) {
```

Comment:
also koa 2 doesn't require 7.6, just recommends it. it just requires async/await support